features/deleting-a-manual.feature

```diff
@@ -16,10 +16,18 @@ Feature: Rake task to delete a manual
     Given a draft manual exists without any sections
     And a draft section exists for the manual
     When I discard the draft manual
+    And I confirm draft deletion
     Then the manual and its sections are deleted
 
   Scenario: Deleting a published manual
     Given a published manual exists
     When I run the deletion script
     Then the script raises an error
     And the manual and its sections still exist
+
+  Scenario: UI safeguarding against deleting a published manual
+    Given a published manual exists
+    And I am on the show page for that manual
+    Then I should not see the discard draft button
+    And I visit the confirm discard page directly
+    Then I am on the show page for that manual
```

Comment:
Feels like this should perhaps be two different scenarios? One testing the button visibility and another the direct access attempt? I might even go further and question whether a lower-level test might be appropriate here as feature tests are quite expensive (assuming they are running via a browser), so I tend to prefer them for happy path only tests. But that may not be practical here and obviously that's a very subjective point

Reply:
I tend to agree with everything you've said, there just aren't much in the way of controller/view specs in the manuals publisher codebase. However, we have taken a stab at introducing some, and removed the cucumber test. Let us know your thoughts! 😄